lesson_11/arrays_ts/src/lesson11.ts

```diff
@@ -3,13 +3,28 @@
  * https://leetcode.com/problems/concatenation-of-array
  */
 export function getConcatenation(nums: number[]): number[] {
-  return [];
+  const ans: number[] = [];
+  for (const num of nums) {
+    ans.push(num);
+  }
+  for (const num of nums) {
+    ans.push(num);
+  }
+  return ans;
 }
 
 /**
  * Provide the solution to LeetCode 2942 here:
  * https://leetcode.com/problems/find-words-containing-character/
  */
 export function findWordsContaining(words: string[], x: string): number[] {
-  return [];
+  const num: number[] = [];
+  for (const word of words) {
+    const wordArr = word.split('');
+    const containsX = wordArr.some((char) => x.includes(char));
```

Comment:
Could have just done `wordArr.some(char => x === char)` or as I demonstrated in class, `word.indexOf(x) > -1`.